src/components/Main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App.jsx'
+import './index.css'
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+)
```

Comment:
This file is conflicting with your main.jsx, Change the filename to Gallery.jsx and fill it in with your horned beast components and you should be good!